### PR TITLE
[docs] Iconography content

### LINKED ISSUE #1613: [docs] Update Iconography Page
This issue is for updating iconography guidelines to accomplish the following:

- Remove reference to SVG Sprites
- Add guidelines for usage around inline SVG assets

### PR DESCRIPTION
**Pull request by connor-leech**
Closes #1613 

### Changelog
* Changed tab name
* Removed IBM-specific content in H3
* Removed spaces between numbers and px
* Swapped x for ✕

### CODE REVIEW

**Comment by connor-leech** [comment]
@joshblack I did a short copy pass on this yesterday. Just removed the same content you did in that other PR. Can you verify if this is accurate and clarify what needs to be added?

File: src/content/guidelines/iconography/usage.mdx [hunk 1]
```diff
@@ -136,62 +129,19 @@ When used next to text, icons should be center-aligned.
 </Column>
 </Row>
 
-## Usage for developers
+## Developing with icons
 
-### Using SVG sprite (recommended)
+### Using SVG sprite
```
Comment (joshblack) [inline src/content/guidelines/iconography/usage.mdx]:
```suggestion
### SVG Icons
```

File: src/content/guidelines/iconography/usage.mdx [hunk 2]
```diff
@@ -41,19 +34,19 @@ tabs: ['Library', 'Usage', 'Contribution']
 <br />
 <br />
 
-## Usage for designers
+## Designing with icons
 
 ### Sizing
 
-UI icons that appear within Carbon components are generally 16 px squares, however icons sized at 20 px, 24 px, and 32 px can be also used within the UI. Icons should only be used at their original sizes and not resized in code.
+Carbon components typically use 16px icons. Icons sized at 20px, 24px, and 32px can be also used within the UI.
```
[the remark this answers is not in the document]
Comment (connor-leech) [reply src/content/guidelines/iconography/usage.mdx]: So it should read "Carbon components typically use icons on 16px artboards. Icons on 20px, 24px, and 32px artboards can also be used within the UI." ?

File: src/content/guidelines/iconography/usage.mdx [hunk 3]
```diff
@@ -41,19 +34,19 @@ tabs: ['Library', 'Usage', 'Contribution']
 <br />
 <br />
 
-## Usage for designers
+## Designing with icons
 
 ### Sizing
 
-UI icons that appear within Carbon components are generally 16 px squares, however icons sized at 20 px, 24 px, and 32 px can be also used within the UI. Icons should only be used at their original sizes and not resized in code.
+Carbon components typically use icons on 16px artboards. Icons on 20px, 24px, and 32px artboards can also be used within the UI.
```
Comment (aagonzales) [inline src/content/guidelines/iconography/usage.mdx]: Maybe add something about being consistent with icons sizes across your product. A single product probably should be using 16, 20, 24, and 32.

Comment (connor-leech) [reply src/content/guidelines/iconography/usage.mdx]: Good point! I'll add and request your review 🙌

File: src/content/guidelines/iconography/usage.mdx [hunk 4]
```diff
@@ -136,62 +129,19 @@ When used next to text, icons should be center-aligned.
...
-If you're unable to use the recommended SVG sprite or `svgxuse`, you can inline SVG directly into your HTML.
+Install `carbon-icons`.
 
-Visit carbon-icons and browse the svg folder for any icons you want to use. (Remember, svg subfolders contain deprecated icons. Don't use these.)
+Full installation details can be found in the Carbon icons [GitHub repo](https://github.com/carbon-design-system/carbon-icons).
 
 ### Main files
 
-Carbon icons ship with two main SVG files that contain different sets of external SVG sprite files:
+`carbon-icons` ships with two main SVG files that contain different sets of external SVG sprite files:
```
Comment (chrisconnors-ibm) [inline src/content/guidelines/iconography/usage.mdx]: remove all the references to 'sprite' or `sprite files`.

change:
`carbon-icons ships with two main SVG files that contain different sets of external SVG sprite files:`
to
`carbon-icons ships with two main SVG files:`

and remove table row:
`sprite.svg | SVG sprite contains legacy icons | 1.x,2.x&3.x
-- | -- | --`

**Pull request closed without merging**